### PR TITLE
Add recording and slides from DebConf16 presentation by James Shubin

### PR DESCRIPTION
**Pull request by jacksgt**
Just watched the recording from DebConf, checked out the repo and saw that the video link was  missing.
Fixed  :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ We'd love to have your patches! Please send them by email, or as a pull request.
 * Felix Frank; blog: [Puppet Powered Mgmt (puppet to mgmt tl;dr)](https://ffrank.github.io/features/2016/06/19/puppet-powered-mgmt/)
 * James Shubin; blog: [Automatic clustering in mgmt](https://ttboj.wordpress.com/2016/06/20/automatic-clustering-in-mgmt/)
 * James Shubin; video: [Recording from CoreOSFest 2016](https://www.youtube.com/watch?v=KVmDCUA42wc&html5=1)
+* James Shubin; video: [Recording from DebConf16: Next Generation Config Mgmt](http://meetings-archive.debian.net/pub/debian-meetings/2016/debconf16/Next_Generation_Config_Mgmt.webm)
```
Comment (purpleidea) [inline README.md]: "Recording from DebConf16" is probably sufficient. It's all about mgmt.

**Comment by purpleidea** [comment]
@jacksgt Great, thank you! Your patch is failing because of the trailing whitespace. Can you put it all on one line, and rebase please?

**Comment by jacksgt** [comment]
Sure, done!

**Comment by purpleidea** [comment]
On Wed, Jul 6, 2016 at 4:02 PM, jacksgt <notifications@github.com> wrote:

> Sure, done!



Looks great, merged thanks!
Please let us know if you'd like to get involved with more patches and you
need some getting started tips or pointers around the code.

Cheers,
James

**Pull request merged**